docs/XX_endringslogg.adoc

```diff
@@ -2,6 +2,11 @@
 
 Mindre redaksjonelle justeringer av tekster (f.eks. egenskapsbeskrivelser, lenker osv.) loggføres ikke.
 
+== Endringer fra versjon 2.1.1 til 2.2
+
+Fra 2.1 til 2.2:
+* Lagt til ny egenskap Datasett: i serie (`dcat:inSeries`), klassen Datsett serie (`dcat:DatasetSeries`) og egenskapene første (`dcat:first`), siste (`dcat:last`) og forrige (`dcat:prev`) som er ny i DCAT 3.  
```

Comment:
1. bør dele punktet i to, første punkt om Datasett og andre punkt om Datasett serie. 
2. dcat:next og dcat:prev mangler omtale av for klassen Datasett.

Reply:
(1) er fiksa i 92be7818bdee90bc2ae5f9423865eb0e45ed83b4 
(2) er problematisk. Eg meiner dcat:prev er dokumentert, men dcat:next er ikkje ein del av dcat 3 vokabularet. Derfor er den ikkje med.

Reply:
Eg foreslår at vi venter med å ta inn dcat:next